src/test/java/pages/GoogleSearchResultsPage.java

```diff
@@ -0,0 +1,42 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindAll;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class GoogleSearchResultsPage extends AbstractPage {
+
+    @FindAll({@FindBy(xpath = "//cite[contains(text(), 'wikipedia')]")})
+    private List<WebElement> generalSearchResults;
+
+    @FindBy(xpath = "//div[@id='rso']/div[2]//cite")
+    private WebElement firstEnglishWikipediaLink;
+
+    protected AbstractPage openPage() {
```

Comment:
public GoogleSearchResultsPage openPage() { return this; }

Reply:
Refactored, thank you.
Please see in the following commit.